src/parser/csql_grammar.y

```diff
@@ -28466,6 +28466,28 @@ pt_check_one_stmt(char* p)
                     }
                 }
           }
+        else if((p[0] == '-' && p[1] == '-') || (p[0] == '/' && p[1] == '/'))
+        {
+                for(p += 2; *p ; p++)
+                {
+                    if (*p == '\n')
+                    {
+                       p++;
+                       break;
+                    }
+                }
+        }
+        else if(p[0] == '/' && p[1] == '*')
+        {
+                for(p += 2; *p ; p++)
+                {
+                    if (p[0] == '*' && p[1] == '/')
```

Comment:
Instead of using a for or if statement,
```
p = strstr(p, "*/");
```
How about using ?

Reply:
That's a good comment.
But in the same vein as the comment above, I'll keep it current.